demo/test/components/previews/primer/docs/layout_preview/default.html.erb

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Might we remove these empty lines in the previews?

Reply:
we don't commit them anymore since https://github.com/primer/view_components/pull/603 :)